packages/clay-charts/src/BillboardWrapper.tsx

```diff
@@ -11,6 +11,9 @@ interface IProps extends ChartOptions {
 	elementProps?: React.HTMLAttributes<HTMLDivElement>;
 }
 
+const useIsomorphicLayoutEffect =
+	typeof window !== 'undefined' ? React.useLayoutEffect : React.useEffect;
```

Comment:
(Non-blocking) nit: a ternary like this is probably more readable without the negation.

```js
const a = b === c ? d : e;
```

vs.

```js
const a = b !== c ? e : d;
```

In this case because `'undefined'` is itself a kind of negation (ie. "not defined"), I think the case is even stronger because it reads like a _double_ negation (ie. "if window is not not defined").

```js
const useIsomorphicLayoutEffect =
	typeof window === 'undefined' ? React.useEffect : React.useLayoutEffect;
```